classes/class-wc-connect-shipping-label.php

```diff
@@ -148,7 +157,25 @@ protected function get_name( WC_Product $product ) {
 		public function get_selected_packages( WC_Order $order ) {
 			$packages = $this->get_packaging_metadata( $order );
 			if ( ! $packages ) {
-				return $this->get_items_as_individual_packages( $order );
+				$items = array();
+				$weight = 0;
+
+				foreach( $this->get_all_items( $order ) as $id => $item ) {
+					$items[ $id ] = $item;
+					$weight += $item[ 'weight' ];
+				}
```

Comment:
What do you think about storing `$this->get_all_items( $order )` in `$items`, and calculating the weight sum without a loop?

Something like.. `$weight = array_sum( wp_list_pluck( $items, 'weight' ) );`